packages/mdc-chips/_variables.scss

```diff
@@ -19,3 +19,9 @@ $mdc-chip-fill-color-default: rgba(black, .08);
 $mdc-chip-ink-color-default: rgba(black, .87);
 $mdc-chip-horizontal-padding: 12px;
 $mdc-chip-vertical-padding: 7px;
+$mdc-chip-with-icon-padding: 4px;
+
+$mdc-chip-icon-color: rgba(0, 0, 0, .54);
+$mdc-chip-trailing-icon-size: 18px;
+$mdc-chip-trailing-icon-hover: rgba(0, 0, 0, .62);
+$mdc-chip-trailing-icon-focus:rgba(0, 0, 0, .87);
```

Comment:
Missing space after `:` here. Also, add `-color` to the hover/focus variable names?